src/algorithms/machinelearning/tensorflowpredictmusicnn.h

```diff
@@ -53,7 +53,8 @@ class TensorflowPredictMusiCNN : public AlgorithmComposite {
   ~TensorflowPredictMusiCNN();
 
   void declareParameters() {
-    declareParameter("graphFilename", "the name of the file containing the model to use", "", Parameter::STRING);
+    declareParameter("graphFilename", "the name of the file from which to load the TensorFlow graph", "", "");
```

Comment:
Any particular reason for `""`? In many other algorithms we are using Parameter::STRING:

```
./src/algorithms/highlevel/gaiatransform.h:    declareParameter("history", "gaia2 history filename", "", Parameter::STRING);
./src/algorithms/io/monowriter.h:    declareParameter("filename", "the name of the encoded file", "", Parameter::STRING);
./src/algorithms/io/monowriter.h:    declareParameter("filename", "the name of the encoded file", "", Parameter::STRING);
./src/algorithms/io/yamlinput.h:    declareParameter("filename", "Input filename", "", Parameter::STRING);
./src/algorithms/io/audioloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/audioloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/monoloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/monoloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/metadatareader.h:    declareParameter("filename", "the name of the file from which to read the tags", "", Parameter::STRING);
./src/algorithms/io/audiowriter.h:    declareParameter("filename", "the name of the encoded file", "", Parameter::STRING);
./src/algorithms/io/audiowriter.h:    declareParameter("filename", "the name of the encoded file", "", Parameter::STRING);
./src/algorithms/io/eqloudloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/eqloudloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/easyloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/io/easyloader.h:    declareParameter("filename", "the name of the file from which to read", "", Parameter::STRING);
./src/algorithms/extractor/freesoundextractor.h:    declareParameter("profile", "profile filename. If specified, default parameter values are overwritten by values in the profile yaml file. If not specified (empty string), use values configured by user like in other normal algorithms", "", Parameter::STRING);
./src/algorithms/extractor/musicextractor.h:    declareParameter("profile", "profile filename. If specified, default parameter values are overwritten by values in the profile yaml file. If not specified (empty string), use values configured by user like in other normal algorithms", "", Parameter::STRING);
```

Reply:
They do different things:
-  `Parameter::STRING` means that by default the variable is not initialized
- `""` means that by default the variable is initialized to an empty string

In #1161 we modified the logic of `TensorflowPredict` so that the configuration is skipped when the parameters  `graphFilename` and `savedModel` are `""`, mainly  facilitate reusing the logic in `TensorflowPredict` instead of reimplementing it in dependent algorithms (e.g., `PitchCrepe`, `TensorflowPredictMusiCNN`...)